conf/ocsci/io_in_bg.yaml

```diff
@@ -1,4 +0,0 @@
----
-# This config file will enable IO to be running in the background
-RUN:
-  io_in_bg: True
```

Comment:
Is this unrelated cleanup?

Reply:
yes, it's unrelated. We don't need this file anymore

Reply:
Should have been in a different PR/commit.